scalameta/src/main/scala/scala/meta/internal/parsers/Parsers.scala

```diff
@@ -2724,7 +2784,9 @@ private[meta] class Parser(val input: Input)(implicit val dialect: Dialect) { pa
    */
   def templateParents(): List[Term] = {
     val parents = new ListBuffer[Term]
-    def readAppliedParent() = parents += constructorCall(startModType())
+    def readAppliedParent() =
+      if (token.is[Ellipsis]) parents += ellipsis(1, unquote[Term])
```

Comment:
`Ctor.Call`, of course. Did that change right before implementing `Ctor.Call` and forgot to review.